cpg_workflows/large_cohort/dense_subset.py

```diff
@@ -15,7 +15,7 @@ def run(
     and return a dense MatrixTable with split multiallelics.
     @return: filtered and densified MatrixTable.
     """
-    if can_reuse(out_dense_mt_path):
+    if can_reuse(out_dense_mt_path, overwrite=True):
```

Comment:
@violetbrina Why does `can_reuse` have these weird semantics?

https://github.com/populationgenomics/production-pipelines/blob/f25a8086a4f1756495faa64c1ace37df694e4851/cpg_workflows/utils.py#L70-L81

It seems odd not having to specify an `overwrite` parameter (as that will _always_ return `False`), so why is it an optional parameter? And should the value of `overwrite` within the context of `production-pipelines` come from a config value instead of hardcoding it (like setting it to `True` here)?

Reply:
An excellent question. Yeah probably should be configurable. I've had a look through the code and there doesn't seem to be a clear answer. I guess it's worth working out if we should consider this scenario for the metamist/production pipelines integration. Instead of doing hard file checks like this I guess we should ask metamist if a file should be reused or not and control the logic on that end.

Happy to approve for now in order for Kat to get this done.